network db rewrite (#12534)
containers/podman: docs/source/markdown/podman-run.1.md

#### **--network**=*mode*, **--net**

> Set the network mode for the container. Invalid if using **--dns**, **--dns-opt**, or **--dns-search** with **--network** that is set to **none** or **container:**_id_. If used together with **--pod**, the container will not join the pod's network namespace.

Reviewer: ```suggestion
Set the network mode for the container. Invalid if using **--dns**, **--dns-opt**, or **--dns-search** with **--network** that is set to **none** or **container:**_id_. If used together with **--pod**, the container will not join the pod's network namespace.
```
```suggestion
Set the network mode for the container. Invalid if using **--dns**, **--dns-opt**, or **--dns-search** with **--network**  set to **none** or **container:**_id_. If used together with **--pod**, the container will not join the pod's network namespace.
```

Valid _mode_ values are:

- **bridge[:OPTIONS,...]**: Create a network stack on the default bridge. This is the default for rootfull containers. It is possible to specify these additional options:
  - **alias=name**: Add network-scoped alias for the container.
  - **ip=IPv4**: Specify a static ipv4 address for this container.
  - **ip=IPv6**: Specify a static ipv6 address for this container.
  - **mac=MAC**: Specify a static mac address address for this container.
  - **interface_name**: Specify a name for the created network interface inside the container.

> - **\<network name or ID\>[:OPTIONS,...]**: Connect to a user-defined network, this is just the network name or ID from a network created by **[podman network create](podman-network-create.1.md)**. Using the network name implies the network mode bridge. It is possible to specify same options as described under bridge above. You can use **--network** multiple times to specify additional networks.

Reviewer: if you use comments from above, use them here too.

#### **--network-alias**=*alias*

> Add network-scoped alias for the container. This will set the alias for all networks that the container joins. To set a name only for a specific network use the alias option as described under **--network**.

Reviewer: ```suggestion
Add network-scoped alias for the container, setting the alias for all networks that the container joins. To set a name only for a specific network, use the alias option described under **--network**.
```
